cryosparc/command.py

```diff
@@ -18,6 +18,21 @@
 RETRY_INTERVAL = int(os.getenv("CRYOSPARC_COMMAND_RETRY_SECONDS", 30))
 
 
+class CommandError(Exception):
```

Comment:
if this will be imported by our other repos, it may be handy to put this and other shared types into a `types.py` file

Reply:
Moved this and some other error classes to errors.py, there aren't many other types that may be imported elsewhere